2023-01-17 15:46:29,564 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=5)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-02-16 15:19:54,644 - root - ERROR - 
Traceback (most recent call last):
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 69, in dump_data
    page_1_text, page_2_text = extract_text(file_content)
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 43, in extract_text
    reader = PyPDF2.PdfReader(file_content, strict=False)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 317, in __init__
    self.read(stream)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1409, in read
    self._find_eof_marker(stream)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1465, in _find_eof_marker
    raise PdfReadError("EOF marker not found")
PyPDF2.errors.PdfReadError: EOF marker not found
2023-02-18 15:18:17,364 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-02-18 15:18:24,369 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-02-18 15:18:25,835 - root - ERROR - 
Traceback (most recent call last):
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 69, in dump_data
    page_1_text, page_2_text = extract_text(file_content)
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 43, in extract_text
    reader = PyPDF2.PdfReader(file_content, strict=False)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 317, in __init__
    self.read(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1409, in read
    self._find_eof_marker(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1465, in _find_eof_marker
    raise PdfReadError("EOF marker not found")
PyPDF2.errors.PdfReadError: EOF marker not found
2023-02-19 15:17:55,582 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-02-19 15:18:02,468 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-02-19 15:18:03,752 - root - ERROR - 
Traceback (most recent call last):
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 81, in dump_data
    reader = PyPDF2.PdfReader(file_content, strict=False)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 317, in __init__
    self.read(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1409, in read
    self._find_eof_marker(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1465, in _find_eof_marker
    raise PdfReadError("EOF marker not found")
PyPDF2.errors.PdfReadError: EOF marker not found
2023-03-01 15:20:51,008 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-03-01 15:20:52,461 - root - ERROR - 
Traceback (most recent call last):
  File "/home/runner/work/sbi_forex_rates/sbi_forex_rates/main.py", line 81, in dump_data
    reader = PyPDF2.PdfReader(file_content, strict=False)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 317, in __init__
    self.read(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1409, in read
    self._find_eof_marker(stream)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1465, in _find_eof_marker
    raise PdfReadError("EOF marker not found")
PyPDF2.errors.PdfReadError: EOF marker not found
2023-03-23 15:18:15,146 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-04-03 16:37:34,434 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-04-03 16:37:41,470 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-04-06 10:30:53,895 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-04-18 10:32:02,913 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-05-04 16:44:20,229 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-05-27 10:31:08,993 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-15 10:32:29,352 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-15 10:32:46,085 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-15 10:33:08,839 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-21 16:36:00,947 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f098ec54910>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-21 16:36:16,979 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f098ec54160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-21 16:36:39,030 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f098ec544c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-06-21 16:37:13,078 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f098ec542e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-03 10:36:14,092 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-03 16:39:12,167 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-03 16:39:28,892 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-05 10:38:47,338 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-08 16:37:56,916 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-10 10:37:51,780 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-10 10:38:08,590 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-07-17 10:37:29,806 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-08-31 10:32:20,638 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1b76734910>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-08-31 10:32:36,673 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1b76734160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-08-31 10:32:58,706 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1b767344c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-08-31 10:33:32,753 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1b767342e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-08-31 10:34:30,777 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1b76734460>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-13 16:34:51,715 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-13 16:34:58,812 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-16 16:32:36,456 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-16 16:32:43,637 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-19 16:36:56,243 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-19 16:37:03,176 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-20 16:36:15,447 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-20 16:36:22,551 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-20 16:36:35,650 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-20 16:37:00,757 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-22 16:35:20,369 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-22 16:35:27,538 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-22 16:35:40,723 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-23 16:32:00,320 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-23 16:32:07,511 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-23 16:32:20,703 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-23 16:32:45,913 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-24 16:32:25,861 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-09-24 16:32:33,053 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-10-02 10:33:05,237 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-10-02 10:33:22,055 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-10-05 10:34:20,942 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-11-06 10:35:31,542 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-06 16:38:10,345 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-11-06 16:38:12,901 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-07 10:32:59,838 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-07 16:36:53,863 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-08 10:33:18,261 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-08 16:36:30,790 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-10 10:32:42,874 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-10 16:35:52,801 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-11 10:30:50,536 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-11 16:33:17,492 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-12 10:31:02,479 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-12 16:32:51,771 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-17 10:34:10,949 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-11-17 10:34:13,499 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-17 16:39:31,535 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-18 10:31:17,797 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-19 10:31:35,626 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-19 16:33:56,155 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-20 10:35:40,866 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-20 16:35:08,454 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-21 10:40:21,050 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f790992ad70>, 'Connection to sbi.co.in timed out. (connect timeout=10)')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-11-21 10:40:37,827 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f790992ae30>, 'Connection to sbi.co.in timed out. (connect timeout=10)')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-11-21 10:40:48,073 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-22 10:36:22,613 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='bank.sbi', port=443): Read timed out. (read timeout=10)")': /
2023-11-22 10:36:39,308 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='bank.sbi', port=443): Read timed out. (read timeout=10)")': /
2023-11-22 10:37:02,084 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='bank.sbi', port=443): Read timed out. (read timeout=10)")': /
2023-11-22 10:37:36,857 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='bank.sbi', port=443): Read timed out. (read timeout=10)")': /
2023-11-22 10:37:47,533 - root - INFO - Invalid PDF. Response MIME type seems to be HTML document, ASCII text, with CRLF, LF line terminators
2023-11-22 10:37:47,533 - root - INFO - Using a proxy to retry getting the PDF
2023-11-22 10:38:14,579 - root - INFO - Found a valid PDF in 1 try. Breaking away
2023-11-22 10:38:14,692 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-22 16:37:07,077 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-23 10:33:54,922 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-23 16:36:07,450 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-24 10:34:12,798 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-24 16:36:17,169 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-25 10:30:43,096 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-25 16:33:09,221 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-26 10:31:29,375 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-26 16:33:27,412 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-27 10:34:57,700 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-27 16:38:01,495 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-28 10:34:50,450 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-28 16:38:05,507 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-29 10:34:43,488 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-29 16:37:58,191 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-30 10:33:56,768 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-11-30 16:38:35,555 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-01 10:34:31,491 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-01 16:36:26,542 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-02 10:30:41,405 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-02 16:33:16,125 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-03 10:30:53,860 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-03 16:34:23,851 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-04 10:35:56,450 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-04 16:36:37,167 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-05 10:35:19,002 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:35:35,767 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:35:48,766 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:36:13,773 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:37:02,831 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:37:05,256 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:37:12,249 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-05 10:37:25,249 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-06 10:34:36,136 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-06 16:38:29,419 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-07 10:34:50,100 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-07 16:38:31,319 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-08 10:33:41,295 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-08 16:37:08,487 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-09 10:31:04,369 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-09 16:34:02,331 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-10 10:30:57,164 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-10 16:34:09,455 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7113a41300>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-10 16:34:25,484 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7113a414b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-10 16:34:47,519 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7113a41750>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-10 16:35:21,544 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7113a41930>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-10 16:36:19,612 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7113a41b10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-10 16:36:32,354 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-11 10:36:45,793 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-11 16:39:14,264 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f88d0151300>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-11 16:39:30,291 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f88d01514b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-11 16:39:32,496 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-12 10:34:56,277 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-12 16:37:17,788 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-13 10:34:27,413 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-13 16:36:49,298 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-14 10:34:32,624 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-14 16:33:29,782 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-15 10:34:33,965 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-15 16:36:41,945 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-16 10:31:15,933 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-16 16:34:15,786 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-17 10:31:48,094 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-17 16:35:46,326 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-18 10:35:53,521 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-18 16:37:12,594 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-19 10:34:09,817 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-19 16:30:51,362 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-20 10:31:04,747 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-20 16:36:08,221 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-21 10:33:24,360 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-21 16:36:47,076 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-22 10:33:13,519 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-22 16:35:13,184 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-23 10:31:40,240 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-23 16:34:15,617 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-24 10:31:06,237 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f06252051b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2023-12-24 10:31:12,404 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-24 16:33:41,258 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-25 10:32:46,911 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-25 16:35:54,373 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-26 10:32:04,524 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-26 16:36:03,709 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-27 10:32:53,336 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-27 16:36:53,516 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-28 10:32:34,887 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-28 16:36:52,286 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-29 10:33:06,867 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-29 16:36:47,111 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-30 10:30:59,123 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-31 10:31:00,481 - root - ERROR - No data matching the currency regex found
NoneType: None
2023-12-31 16:34:07,897 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-01 10:33:25,048 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-01 16:34:59,984 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-02 10:33:14,683 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-02 16:34:24,092 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-03 10:33:12,025 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-03 16:35:37,072 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-04 10:34:07,493 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-04 16:37:15,264 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-05 10:33:30,029 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-05 16:35:44,320 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-06 10:31:49,507 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-06 16:35:22,714 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-07 10:32:27,559 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-07 16:35:20,928 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-08 10:35:24,493 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-08 16:37:48,690 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-09 10:34:42,757 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-09 16:38:08,734 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-10 10:34:44,191 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-10 16:37:13,183 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-11 10:34:39,546 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-11 16:38:04,749 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-12 10:34:28,255 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-12 16:36:56,059 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-13 10:32:18,930 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-13 16:35:30,758 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-14 10:32:06,470 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-14 16:34:52,693 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-15 10:36:05,365 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-15 16:38:20,868 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-16 10:35:09,444 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-16 16:37:51,298 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-17 10:34:56,462 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-17 16:37:55,908 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-18 10:35:57,665 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-18 16:36:55,144 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-19 10:35:09,034 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-19 16:38:15,656 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-20 10:32:01,807 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-20 16:35:22,866 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-21 10:33:18,765 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-21 16:35:15,475 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-22 10:36:07,427 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-22 16:37:36,435 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-23 10:35:20,534 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-23 16:38:44,983 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-24 10:35:35,798 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-24 16:37:56,587 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-25 10:35:27,742 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-25 16:39:20,414 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-26 10:33:29,871 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-26 16:33:16,997 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-27 10:31:04,724 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd4bd3ea890>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-01-27 10:31:17,524 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-27 16:32:39,885 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-28 10:30:53,560 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-28 16:30:44,094 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-29 10:31:32,556 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-01-29 10:31:35,227 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-29 16:34:01,292 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-30 10:33:06,439 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.sbi.co.in', port=443): Read timed out. (read timeout=10)")': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-01-30 10:33:08,799 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-30 16:35:43,928 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-31 10:31:19,269 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-01-31 16:30:46,593 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-01 10:32:10,913 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-01 16:32:37,357 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-03 10:30:50,244 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-03 16:32:29,327 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-04 10:30:44,752 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-04 16:30:52,476 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-05 10:33:31,691 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-05 16:32:39,321 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-06 10:32:55,888 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-06 16:33:30,994 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-07 10:31:34,926 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-07 16:34:40,678 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-08 10:30:48,813 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-08 16:33:30,718 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-09 10:30:48,532 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-09 16:33:31,795 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-10 10:30:40,738 - root - ERROR - No data matching the currency regex found
NoneType: None
2024-02-15 16:33:15,613 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5570e4130>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-02-15 16:33:31,630 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5570e43a0>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-02-15 16:33:53,653 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5570e4580>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-02-15 16:34:27,689 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5570e4760>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-02-15 16:35:25,749 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5570e4940>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-02-16 06:53:02,095 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f20764fa0>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-18 10:37:56,464 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x78f34396ceb0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-18 10:38:12,569 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x78f34396d120>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-18 10:38:34,668 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x78f34396d300>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-18 10:39:08,714 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x78f34396d4e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-24 16:38:01,974 - urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f56a1874b20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
2024-03-24 16:38:23,021 - urllib3.connectionpool - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f56a1874d90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /documents/16012/1400784/FOREX_CARD_RATES.pdf
